osx/spec/localhost/android_spec.rb

```diff
@@ -0,0 +1,12 @@
+require 'spec_helper'
+
+describe 'android' do
+  %w(
+    java
+    android-studio
+  ).each do |pkg|
+    describe package(pkg) do
+      it { should be_installed.by('homebrew_cask') }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.